src/components/DateTime/DateTimePicker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from './DateTimePicker.module.css';
+import classNames from "classnames";
+import { Label } from "../Typography";
+import { FormElement } from "../FormElement";
+
+export type DateTimePickerProps = {
+    label?: string;
+    dateTimeValue?: any;
```

Comment:
It’s better to not use the `any` type. Can this value really be anything? Or doesn’t react types spec for HTML define valid types? See this screenshot from react's types:

<img width="681" alt="Screen Shot 2022-11-14 at 4 32 14 PM" src="https://user-images.githubusercontent.com/2483476/201769964-bf4384e2-a984-4871-8260-c56df96e05c6.png">

What would be an appropriate type for this property?

Reply:
Changed type to string instead of any